scipy/signal/signaltools.py

```diff
@@ -1707,20 +1707,20 @@ def resample(x, num, t=None, axis=0, window=None):
     sample of the next cycle:
 
     >>> from scipy import signal
-    
+    >>> import numpy as np
```

Comment:
this isn't needed, can you remove it? the numpy import is assumed by all scipy examples; we don't want to put in this line hundreds of times